packages/workflow/src/utils/authUtils.ts

```diff
@@ -10,6 +10,7 @@
  */
 import * as decode from 'jwt-decode'
 import * as Hapi from '@hapi/hapi'
+import { hasScope } from '@workflow/../../commons/build/dist/types'
```

Comment:
This looks a bit funny. Maybe your editor added it? I could probably be just `'@opencrvs/commons/types'`

Reply:
yes!